### ISSUE TITLE
how to train cityscape with a single 1080ti?

### ISSUE THREAD

**Issue by Jiakui** [speedinghzl/pytorch-segmentation-toolbox#25]
It seems that one 1080 ti will encounter the out of memory error. How to make it work? Can we just reduce the batch size or image size ?

Thanks!

**Comment by speedinghzl** [comment]
Of course, you can reduce the batch size or image size to run this repo on a 1080ti. But I think it will result in a bad result.
